src/frontends/paddle/src/op/linspace.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/frontend/paddle/visibility.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs linspace(const NodeContext& node) {
+    auto start = node.get_input("Start");
+    auto stop = node.get_input("Stop");
+    auto num = node.get_input("Num");
+
+    // compute delta value, i.e. distance between neighbor values of the result
+    auto const_one = std::make_shared<default_opset::Constant>(start.get_element_type(), Shape{}, 1);
+    Output<Node> num_minus_one = std::make_shared<default_opset::Subtract>(num, const_one);
+    num_minus_one = std::make_shared<default_opset::Convert>(num_minus_one, start.get_element_type());
+    Output<Node> delta = std::make_shared<default_opset::Subtract>(stop, start);
+    delta = std::make_shared<default_opset::Divide>(delta, num_minus_one);
+
+    // generate a range of numbers [0, 1, ..., num)
+    auto const_zero = std::make_shared<default_opset::Constant>(start.get_element_type(), Shape{}, 0); 
+    auto const_num = std::make_shared<default_opset::Squeeze>(num);
+    auto range0_n = std::make_shared<default_opset::Range>(const_zero, const_num, const_one, start.get_element_type());
```

Comment:
I just wonder can we use ```range``` for a direct output ? so we do not need ```Multiply``` and ```Add```, just a guess

Reply:
great idea, it is important to simplify the operator code. but the behavior of `Range` is subtly different from `Linspace` even though their input parameters are similar with each other. Using `Range` for a direct output maybe require the overal re-design of `linspace`.

Reply:
> subtly

Thanks, could you clarify what's gap between them? can we convert the ```num``` in Linspace into ```step``` in Range

Reply:
Thanks, could you clarify what's gap between them? can we convert the num in Linspace into step in Range

Reply:
`step` can be calculated by `num`, which has already implemented in line 30 of `linspace.cpp`. The key point is the behavior of `Range`. For instance, same input (start: [0] stop: [1] num: [4]/step: [0.33]), and the result will be:
`Range`: [0, 0.33, 0.66]
`Linspace`: [0, 0.33, 0.66, 1]
And the input of Range must be scalar, which means <Squeeze> must be taken before range operator.
![image](https://user-images.githubusercontent.com/69072522/221104981-71fec25f-b471-4aa1-aeea-f87e0abc2b47.png)
I think it will become more complicate after adjusting the code.

Reply:
Great thanks for your illustration, since its a advanced task or our target is to find out a solution with best inference performance. Could you help to try both solution with [benchmark_app](https://github.com/openvinotoolkit/openvino/tree/master/tools/benchmark_tool) to compare their performance ?
any suggestion from your perspective @ceciliapeng2011 ?

Reply:
Can ```start``` and ```stop``` be negative here, and can you put this example ( (start: [0] stop: [1] num: [4]/step: [0.33])) into your test case, thanks?

Reply:
OK，test samples had been added